### DISCUSSION TITLE
Orchid Quill Add Table

### DISCUSSION THREAD

**Discussion by safeandroidtv** [orchidsoftware/platform#1893] [Q&A]
Hello everyone,

I am trying to add tables to my Quill editor from:
[https://github.com/volser/quill-table-ui](https://github.com/volser/quill-table-ui)

What I did:

1. **/config/platform.php**
Added scripts

```
    'resource' => [
        'stylesheets' => [
            '/css/quill-table-ui.css',
        ],
        'scripts' => [
            'https://unpkg.com/quill-table-ui@1.0.5/dist/umd/index.js',
            '/js/quill-table-ui.js',
        ],
    ],
```

2. File contents - **quill-table-ui.js**:

```
document.addEventListener('orchid:quill', (event) => {

    event.detail.quill.register("modules/tableUI", QuillTableUI.default, true);

    event.detail.options.modules = {
    	table: true,
    	tableUI: true
    }
});
```

But I didn't get anything, what am I doing wrong? How to combine Orchid and this module?

**Comment by safeandroidtv** [comment]
Help please...

**Reply by tabuna** [reply]
I see that this is a [library](https://github.com/volser/quill-table-ui) for quill 2.0 which is available from beta. The orchid package uses the stable version.

**Reply by Hallecsy** [reply]
@safeandroidtv did you solve the problem?

**Reply by shipinov73** [reply]
@tabuna Hi! Is there any updates about integration tables into Orchid Quill Editor?
